f8a_report/manifest_main.py

```diff
@@ -0,0 +1,28 @@
+"""Dynamic Manifest Generation."""
+
+import logging
+from datetime import datetime as dt, timedelta
+from report_helper import ReportHelper
+from manifest_helper import manifest_interface
+import os
+
+logger = logging.getLogger(__file__)
+
+
+def main():
+    """Generate the dynamic manifest file weekly."""
+    r = ReportHelper()
+    today = dt.today()
+
+    start_date_wk = (today - timedelta(days=7)).strftime('%Y-%m-%d')
+    end_date_wk = today.strftime('%Y-%m-%d')
+
+    logger.info(os.environ.get('GENERATE_MANIFESTS', 'False'))
```

Comment:
Can you more info into this logging statement?

Reply:
Ok